src/OpenTelemetry.Shims.OpenTracing/ActivityContextExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// <copyright file="ActivityContextExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+
+namespace OpenTelemetry.Shims.OpenTracing
+{
+    public static class ActivityContextExtensions
+    {
+        private static readonly string InvalidSpanId = default(ActivitySpanId).ToHexString();
+        private static readonly string InvalidTraceId = default(ActivityTraceId).ToHexString();
+
+        public static bool IsValid(this ActivityContext activityContext)
+        {
+            bool success = true;
+
+            var traceId = activityContext.TraceId.ToHexString();
```

Comment:
we should avoid this Tostring part. we can compare the traceid,spanid directly with their defaults.

Reply:
(activityContext.TraceId != defaultTraceId) || (activityContext.SpanId != defaultSpanId).

defaultTraceId = default(ActivityTraceId).

Reply:
This extension is no more required. Removing it.